config/routes.rb

```diff
@@ -280,7 +280,6 @@
     get '/backup_code_refreshed' => 'users/backup_code_setup#refreshed'
     get '/backup_code_reminder' => 'users/backup_code_setup#reminder'
     get '/backup_code_setup' => 'users/backup_code_setup#index'
-    patch '/backup_code_setup' => 'users/backup_code_setup#create', as: :backup_code_create
```

Comment:
It looks like we still have something directing to this page, as production logs show activity for `PATCH /backup_code_setup`. I'll investigate.

Reply:
This should be fixed as of #10464, but will confirm once those changes go live in RC 374.